overrides/config/betterquesting/DefaultQuests.json

```diff
@@ -35800,7 +35800,7 @@
+          "desc:8": "Combine a Corrosive Will Crystal, Blutonium Dust and some Concentrated Cloud Seed in the Vat to obtain some Growth Infusion Liquid! Then use your Growth Infusion Liquid to obtain some Crop Sticks!\n\nThese sticks are way more powerful than they look like - they allow you to duplicate any Mystical Agriculture Seed planted on them, either through spreading or using the Clipper.\n\nYou can also mutate the same type of crop with itself by planting 2 of them on a Crop Stick placed diagonally from each other, and placing a double-Crop Stick in the corner. The Seeds will eventually mutate (which you can speed up a lot with the Imaginary Time Block), creating a potentially better one.\n\n§c§lWARNING:§r Do NOT use an EnderIO Farming Station with Plants on Crop Sticks. It will CRASH your game! Instead, try any of the following to harvest the crops:\n\n- Botania Drum of the Wild or Horn of the Wild inside a Player Simulator;\n- An Actually Additions Farmer;\n- Or an Industrial Foregoing Plant Interactor.\n\nIn all cases, you\u0027ll also need a way to gather the resources. I recommend a Ranged Collector or an Absorption Hopper.\n\nEach AgriCraft Seed has 3 stats, and maxing them out is commonly referred to as \"10/10/10\u0027ing the Seed\". The first stat increases the rate at which the crop grows and the second one increases the average yield of the crop. The third stat is not important for weeds since they are disabled in this pack but it also increases the chance of getting clippings from clippers.\n\nYou can check the 10/10/10\u0027ing progress in a Seed Analyzer.\n\nYou\u0027ll need a bunch of Seeds in one of the later Chapters, so definitely take advantage of AgriCraft!"
```

Comment:
having a `1, 2. 3` syntax leads to it being stilted, i suggest slightly altering the text and using semicolons:
The first stat increases the rate at which the crop grows; the second stat increases the average yield of the crop; the third stat normally reduces the damage weeds can do, but since weeds are disabled in this pack, it only increases the chance of getting clippings from clippers.